src/components/Layout/Container.jsx

```diff
@@ -8,16 +8,23 @@ const Container = ({
   style,
   className,
   topBannerHeight,
+  headerHeight,
+  gapBetweenBannerAndMain,
```

Comment:
I think this should be renamed to something like: `bannerAndMainSpacing`

Reply:
maybe `bannerAndMainGap` I think `gap` is more used term than `spacing` in grid systems, what do you think ?